frontend/packages/topology/src/components/quick-search/utils/quick-search-utils.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+import { CatalogItem } from '@console/plugin-sdk';
+import { keywordCompare } from '@console/dev-console/src/components/catalog/utils/catalog-utils';
+import { QuickStart } from '@console/app/src/components/quick-starts/utils/quick-start-types';
+import { Text, TextVariants } from '@patternfly/react-core';
+
+export const quickSearch = (items: CatalogItem[], query: string) => {
+  return items.filter((item) => keywordCompare(query, item));
+};
+
+export const getTransformedQuickStarts = (
+  quickStarts: QuickStart[],
+  setActiveQuickStart: (quickStartId: string, totalTasks?: number) => void,
+): CatalogItem[] => {
+  return quickStarts.map((qs: QuickStart) => {
+    const description = (
+      <>
+        <Text component={TextVariants.p}>{qs.spec.description}</Text>
+        {qs.spec.prerequisites?.length > 0 && (
+          <>
+            <Text component={TextVariants.h6}>Prerequisites</Text>
+            {qs.spec.prerequisites.map((pq: string) => (
+              <Text component={TextVariants.p}>{pq}</Text>
+            ))}
+          </>
+        )}
+      </>
+    );
```

Comment:
I think we need to update this to the new prerequisites design for quick starts. I added a comment on the design doc regarding this and got the answer from UX that we should keep it consistent. 

I think we can directly reuse `QuickStartTileDescription` component here.

```suggestion
    const description = <QuickStartTileDescription description={qs.spec.description} prerequisites={qs.spec.prerequisites} />
```

Reply:
The reason for the new design in the quick start tile is because of limited available space. In the catalog sidebar we don't have this restriction.

Whatever UX decides on I guess.

Reply:
Yeah, that why I asked this question in the design docs and they agreed to make it consistent everywhere.